data/json/items/obsolete.json

```diff
@@ -531,5 +531,37 @@
         "loudness": 0,
         "count" : 10,
         "effects" : ["RECOVER_3", "NEVER_MISFIRES"]
-    }
+    },{
+    "id": "acidbomb",
+    "type": "TOOL",
+    "symbol": "*",
+    "color": "yellow",
+    "name": "acid bomb",
+    "description": "This is a fragile container filled with acid.  Throw it to spill out a pool of potent acid.",
+    "price": 5000,
+    "material": "glass",
+    "flags": "ACT_ON_RANGED_HIT",
+    "weight": 650,
+    "volume": 2,
+    "bashing": 4,
+    "to_hit": -1,
+    "category": "weapons",
+    "use_action": "ACIDBOMB_ACT"
+  },{
+    "id": "acidbomb_act",
+    "type": "TOOL",
+    "symbol": "*",
+    "color": "yellow",
+    "name": "acid bomb",
+    "name_plural": "acid bombs",
+    "description": "This is a glass vial, with two chemicals mixing inside.  The vial feels colder and the contents are frothing.  If this mixture is exposed to air (as could happen if you throw the vial), it will spill out as a pool of potent acid.",
+    "price": 0,
+    "material": "glass",
+    "weight": 1268,
+    "volume": 2,
+    "bashing": 4,
+    "to_hit": -1,
+    "category": "weapons",
+    "use_action": "ACIDBOMB_ACT"
```

Comment:
You just removed the iuse function (see item_factory.cpp), items can't use it anymore.